### DISCUSSION TITLE
Semi - Geyser Compatibility[FEATURE]

### DISCUSSION THREAD

**Discussion by SlimeCody** [AddstarMC/Prism-Bukkit#246] [Ideas]
**Is your feature request related to a problem? Please describe.**
Prism already works rather well to process there data being tracked, being able to use the *playername

**Describe the solution you'd like**
A slight change in the syntax would allow us to be able to rollback griefs etc done by players using the Geyser cross compatibility
**Describe alternatives you've considered**
**Additional context**
![image](https://user-images.githubusercontent.com/49579480/104790534-13299780-575d-11eb-83cd-3f2a53cbea94.png)

**Comment by Narimm** [comment]
Just to be clear This doesnt really outline how a player object is perceived in geyser and syntax.  In terms of the naming schemas available...
